guidelines/github-contribution-guide.md

```diff
@@ -0,0 +1,268 @@
+you can now open a pull request against the original Hyperledger repository from which you created your fork and begin the code review process.
+
+- Navigate to https://github.com/hyperledger/<original_repository>
+- Select the Pull Request tab at the top of the page
+- In the top right corner of the Pull Requests page, select New Pull Request
+- On the Compare Changes page, select "compare across forks" at the top of the page
+- Select the Hyperledger repository and branch that you want to merge into as the "base repository" and "base" branch
+- Select your fork and your feature branch as the "head repository" and "compare" branch
+- Select Create Pull Request
+- You'll have the option to open your pull request as a Draft if you are not yet ready to have it reviewed.
+
+Congratulations, you have now submitted your first pull request to a Hyperledger project.
+Your pull request will now run through CI checks.
+You can monitor your pull request CI progress by navigating to the Checks tab of the pull request.
+If there are CI check failures the pull request may not get reviewed and merged.
+Either resolve the failures by updating the pull request, or add a comment to the pull request to request help resolving the failure.
+
+## Updating a Pull Request
+
+As you receive review comments on your pull request (or if pull request checks fail), you may need to make edits to your commit.
+In the local branch you are working from, you may make changes, amend the commit, and re-push to your remote feature branch.
+
+```
+git add -p
+git commit --amend
+git push origin <feature_branch_name> -f
+```
+
+This will update the pull request and re-trigger checks.
+
+If you have created multiple commits locally, it is typically best to squash them into a single final commit.
+For example to squash your two most recent commits into a single commit before pushing:
+
+```
+git rebase -i HEAD~2
+```
+
+This will open an interactive dialog. Change the second (and any subsequent) commit action from \'pick\' to \'squash\' in the dialog.
+The dialog will then present all the commit messages, which you can edit into a final message.
+
+## Cherry-picking your pull request to other branches
+
+After your PR is merged into the main branch, you need to consider whether it should be backported to earlier branches.
+If the content is a new feature designated for the next release, obviously backporting is not appropriate.
+But if it is a fix or update to an existing topic, don't forget to cherry-pick the PR back to earlier branches as needed.
+When in doubt, consult the maintainer that merged the PR for advice.
+Both parties should consider the backport and either party can trigger it.
+You can use the GitHub cherry-pick command, or if the repository has the Mergify app enabled,
+you can simply paste the following command as a comment in your PR after it is merged:
+
+```
+@Mergifyio backport <release-branch>
```

Comment:
Q: Is this enabled on all the repositories across the org?

Reply:
No, I've added language "if the repository has the Mergify app enabled..."